lib/Parse/ParseIfConfig.cpp

```diff
@@ -623,8 +623,18 @@ ParserResult<IfConfigDecl> Parser::parseIfConfig(
     foundActive |= isActive;
 
     if (!Tok.isAtStartOfLine() && Tok.isNot(tok::eof)) {
-      diagnose(Tok.getLoc(),
```

Comment:
Small tips: `Parser` has `diagnose(Token Tok, Diagnostic Diag)` function. You don't need `.getLoc()` here.

Reply:
fixed